feat: Add Column component and Gap and Margin utilities for vertical spacing (#1083)
Amsterdam/design-system: storybook/storybook-react/src/Column/Column.docs.mdx

import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
import * as ColumnStories from "./Column.stories.tsx";
import README from "../../../../packages/css/src/components/column/README.md?raw";

<Meta of={ColumnStories} />

<Markdown>{README}</Markdown>

## Design

> The five [spacing](?path=/docs/docs-design-guidelines-spacing--docs) lengths are available for the length of the gap.

Reviewer: ```suggestion
The five [spacing](?path=/docs/docs-design-guidelines-spacing--docs) sizes are available for the size of the gap.
```

## How to Use

Wrap a Column around a set of components that need the same amount of whitespace between them.

> To add whitespace below a single element, using the utility classes for bottom margins is an alternative, e.g. `class="amsterdam-mb-md`.

Reviewer: ```suggestion
To add whitespace below a single element, you can alternatively use a utility class for bottom margins, e.g. `class="amsterdam-mb-md`.
```
